.github/workflows/Analyze docker images.yml

```diff
@@ -8,6 +8,7 @@ name: Analyze Docker Images
 on:
   pull_request:
     branches: [main, 'b-v*-patch*','feature-*']
+    types: [opened, synchronize, reopened, unlabeled]
```

Comment:
This triggers the workflow when a label gets removed from the PR.
Note that when the `type` is not specified, the default value is `opened, synchronize, reopened`.

Reply:
The use of the labels is very clever to run the workflows after making a commit to an ODS PR, but I think we should only focus on the Initdev and the Dependencies Security Scan, so the default value is ok for the docker actions

Reply:
I reverted the `types` definition for the Docker workflows, thanks.